src/components/dialog-full-screen/dialog-full-screen.test.tsx

```diff
@@ -0,0 +1,432 @@
+import React, { useState } from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import StyledFullScreenHeading from "../../__internal__/full-screen-heading/full-screen-heading.style";
+import DialogFullScreen from "./dialog-full-screen.component";
+import StyledContent from "./content.style";
+import StyledIconButton from "../icon-button/icon-button.style";
+import { StyledHeader, StyledHeading } from "../heading/heading.style";
+import Form from "../form";
+import { StyledFormContent } from "../form/form.style";
+import CarbonProvider from "../carbon-provider";
+import Logger from "../../__internal__/utils/logger";
+
+// mock Logger.deprecate so that no console warnings occur while running the tests
+const loggerSpy = jest.spyOn(Logger, "deprecate");
```

Comment:
**suggestion**: it seems no warnings are raised if the spy is removed. To avoid potentially masking legitimate deprecation warnings in the future, what do you think about removing this?

```suggestion
```

Reply:
Thanks, I'd just realised in the tests I'm working on now, in a different component, that the `loggerSpy` can be removed without warnings appearing in the console - which surprised me, but that probably means it's unneeded here too 👍

Note that I don't agree with the point about "masking legitimate deprecation warnings in the future" - such deprecation warnings still shouldn't appear in the console output when running the tests, as they're not useful there and just add noise to the output (which was why these mocks were added in the first place). But I agree it's better not to mock if it's not needed, so I'll remove it.